Formula/c/codecov-cli.rb

```diff
@@ -3,18 +3,18 @@ class CodecovCli < Formula
 
   desc "Codecov's command-line interface"
   homepage "https://cli.codecov.io/"
-  url "https://files.pythonhosted.org/packages/5c/78/c66189c5f0494f2deb84fe734ee5a5d3b388fa6bfe45cca49bfdef32a6f7/codecov-cli-0.8.0.tar.gz"
-  sha256 "d1dc15353a91fc1664745c6ea936c16d22abf03d23d21ebd42c4efe838cb8551"
+  url "https://files.pythonhosted.org/packages/c4/9e/50af9e41e63a74290a447473a7f7f88af931e3e2b7959486bd30e7e16d80/codecov-cli-9.0.4.tar.gz"
```

Comment:
the version seems a bit off?

Reply:
Wow, I was thought it is 0.9.4

Reply:
![image](https://github.com/user-attachments/assets/89437d23-25ed-46b8-9253-45d308dad5bb)

I should contact to the upstream, Thanks for the catch @chenrui333 !

Reply:
no worries, I feel like they should forfeit the wrong releases.

Reply:
Upstream has chosen to go with this version scheme: https://github.com/Homebrew/homebrew-core/pull/197779#issuecomment-2486937465